src/public/WRLDRouteView.h

```diff
@@ -29,15 +29,41 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void) addToMap;
 
+/*!
+  !Deprecated
+ */
 - (WRLDPolyline*) makeVerticalLine:(WRLDRouteStep*)step
                              floor:(NSInteger)floor
-                            height:(CGFloat)height;
+                            height:(CGFloat)height
+                      isActiveStep:(BOOL)isActiveStep;
 
+/*!
+  !Deprecated
+ */
 - (void) addLinesForRouteStep:(WRLDRouteStep*)step;
 
+/*!
+  !Deprecated
+ */
 - (void) addLinesForFloorTransition:(WRLDRouteStep*)step
                          stepBefore:(WRLDRouteStep*)stepBefore
-                          stepAfter:(WRLDRouteStep*)stepAfter;
+                          stepAfter:(WRLDRouteStep*)stepAfter
+                       isActiveStep:(BOOL)isActiveStep;
```

Comment:
Isn't this a breaking change here? 
Understand that these are deprecated but would prefer that we don't cause a breaking change if possible.